sway/desktop/output.c

```diff
@@ -1027,6 +1028,15 @@ void handle_new_output(struct wl_listener *listener, void *data) {
 
 	transaction_commit_dirty();
 
+	struct sway_output *tmp_output;
+	wl_list_for_each(tmp_output, &root->all_outputs, link) {
+		if (tmp_output == root->fallback_output) {
+			continue;
+		}
+		int width, height;
+		wlr_output_transformed_resolution(tmp_output->wlr_output, &width, &height);
+		wlr_damage_ring_set_bounds(&tmp_output->damage_ring, width, height);
+	}
```

Comment:
Hm. Do we really need to loop over all outputs?

Is it sufficient to move the `wlr_damage_ring_set_bounds()` call above, to a place after `apply_output_config()`?

Reply:
I'll double check 👍

Reply:
You're right! I can now also move the initial logic from the first PR down here :)